.ci/travis/travis.sh

```diff
@@ -494,8 +495,13 @@ no-error-pmd)
   git clone https://github.com/pmd/pmd.git
   cd pmd
   mvn -e install checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}
+  grep "Processing_Errors" $CHECKSTYLE_DIR/target/site/pmd.html | cat > errors.log
+  echo "PMD Processing Errors: "
+  RESULT=$(cat errors.log | wc -l)
+  cat errors.log
```

Comment:
right code , but wrong place
please move it to
https://github.com/checkstyle/checkstyle/blob/09f76443f153cc7ab775c786300684ca392bcf52/.travis.yml#L72
this update will require to move such commands to travis.sh - please do.

Reply:
Ok, I wanted this build to fail first, then I will add  the link to failing travis in comment, then push the fix.